runtime/fresh/middleware.tsx

```
@@ -0,0 +1,183 @@
+      },
+      visibilityOverrides: options.visibilityOverrides,
+    });
+
+  const handler = handlerFor(deco);
+
+  const deepLogFunctions = <T extends unknown>(obj: T, path: string[]): void => {
+    if (typeof obj === "function") {
+      console.log("Function found, path:", path);
+    }
+    if (typeof obj === "object" && obj !== null) {  
+      Object.entries(obj).forEach(([key, value]) => deepLogFunctions(value, [...path, key]));
```

Biome (2.3.13)

[error] This callback passed to forEach() iterable method should not return a value.

Either remove this return or remove the returned value.


(lint/suspicious/useIterableCallbackReturn)